### PR TITLE
Change Output Provider List

### PR DESCRIPTION
**Pull request by andrew-mestas**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@andrew-mestas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-anmest). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@andrew-mestas__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: lib/commands/arm/providers/provider._js [hunk 1]
```diff
@@ -159,3 +168,18 @@ function withClient(wrappedFunction) {
     return wrappedFunction.apply(this, [client].concat(args));
   };
 }
+
+function extractResourceTypesWithLocation(providers, namespace){
+	var provider = providers.filter(function(x){return x.namespace == namespace;})[0];      // Get specific provider
+	var resourceTypes = provider.resourceTypes.map(function(x){return x.resourceType;});    // Get all resources of provider
+    var locations = provider.resourceTypes.map(function(x){return x.locations})				// Get all unique locations of resources 
+							.reduce(function(a,b){return a.concat(b);},[])
```
Comment (amarzavery) [inline lib/commands/arm/providers/provider._js]: indentation

Comment (amarzavery) [inline lib/commands/arm/providers/provider._js]: replace tabs with spaces. indentation size is 2 spaces

**Pull request closed without merging**